src/chromium/policy.md

```diff
@@ -0,0 +1,22 @@
+# Chromium Rust policy
+
+Chromium does not yet allow first-party Rust except in rare cases as approved
+by Chromium's [Area Tech Leads](https://source.chromium.org/chromium/chromium/src/+/main:ATL_OWNERS).
+
+Chromium's policy on third party libraries is outlined [here](https://chromium.googlesource.com/chromium/src/+/main/docs/adding_to_third_party.md#rust) -
+Rust is allowed for third party libraries under various circumstances, including
+if they're the best option for performance or for security.
+
+Very few Rust libraries directly expose a C/C++ API, so that means that nearly
+all such libraries will require a small amount of first-party glue code.
+
+> First-party Rust glue code for a particular third-party crate should
+> normally be kept in `third_party/rust/crate/<crate>/<version>/wrapper`.
```

Comment:
Can you please provide feedback on the violation in the form of `//components/qr_code_generator/*ffi_glue*.rs`?  Please comment in (Google-internal, sorry) https://chat.google.com/room/AAAAk1UCFGg/nnb21XI_FEs

Reply:
In general I think bikeshedding on names and code locations is roughly proportional to the _square_ of the number of participants, and I don't have strong views, so I'll avoid getting involved :) In this course I'm trying to document our current best practice so do let me know if you think I should change this bit!